text/0000-stabilize-marker-freeze.md

```diff
@@ -194,3 +217,4 @@ Note that `core::marker::PhantomData<T>` is `Freeze` regardless of `T`'s `Freeze
 - Adding a `trait Pure: Freeze` which extends the interior immutability guarantee to indirected data could be valuable:
 	- This is however likely to be a fool's errand, as indirections could (for example) be hidden behind keys to global collections. 
 	- Providing such a trait could be left to the ecosystem unless we'd want it to be an auto-trait also (unlikely).
+- `impl !Freeze for T` could be a nice alternative to `PhantomNotFreeze`. However, this would be a significant language change that needs much deeper consideration.
```

Comment:
It's worth mentioning that these are not the same, and so one may not be a straightforward alternative for the other.

Including a `PhantomNotFreeze` field removes the `Freeze` impl for the type.  `impl !Freeze for T` expresses a guarantee that there will *never* be a `Freeze` bound for the type.

Reply:
Right you are, if we stick to the "`impl !Trait for X` means that `impl Trait for X` requires a major bump" interpretation (which is something I'd really enjoy to have), they do indeed yield different results